wurst/objects/abilities/beastmaster/Druid/NaturesBond.wurst

```diff
@@ -0,0 +1,82 @@
+package NaturesBond
+
+// Standard Library Imports:
+import Icons
+import ChannelAbilityPreset
+import BuffObjEditing
+import BuffIds
+import Abilities
+
+// Local Imports:
+import LocalObjectIDs
+import ToolTipsUtils
+import ColorUtils
+import HealingSystem
+import Pets
+
+
+let COOLDOWN = 60.
+let MANA_COST = 20
+let HEAL_DIST = .5
+let DURATION = 15.
+let CAST_RANGE = 15
+let BUTTON_POS_X = 3
+let BUTTON_POS_Y = 2
+let HOTKEY = "D"
+let AOE = 4000
+let NAME = "Nature's Bond"
+let ICON = Icons.bTNSpiritLink
+let TOOLTIP_BASIC = "Nature's Bond"
+let TOOLTIP_EXT   = "The strong bond between the Druid and his pet causes them to split all damage between themselves for "
+    + DURATION.toToolTipOrange() + " seconds. They also share " + (HEAL_DIST*100).toToolTipGreen() + " %".color(HEALTH_COLOR)
+    + " of any healing with eachother."
+    + makeToolTipCooldown(COOLDOWN)
+let BUFF_TOOLTIP = "This unit is sharing damage and healing with another."
+let BUFF_ID = compiletime(BUFF_ID_GEN.next())
+
+@compiletime function createNaturesBond() returns BuffDefinition
+    return new BuffDefinition(BUFF_ID, BuffIds.spiritLink)
+        ..setIcon(ICON)
+        ..setName(1, NAME)
+        ..setArtTarget(1, Abilities.spiritLinkZapTarget)
+        ..setTargetAttachmentPoint0(1, "chest")
+        ..setTooltipNormal(1, "Nature's Bond")
+        ..setTooltipNormalExtended(1, BUFF_TOOLTIP)
+        ..setEditorSuffix(1, "(Wurst)")
+
+@compiletime function createSpiritLinkAbility() returns AbilityDefinitionSpiritLink
+    return new AbilityDefinitionSpiritLink(ABILITY_NATURES_BOND)
```

Comment:
If you want to guarantee a spirit link between the caster and the target, the area of effect must have the same value as the cast range, if not, the spirit link may not find the other target and the casting will be useless. Currently, if you cast spirit link from being afar, it will affect only the pet.